distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -284,6 +284,17 @@ image::catalog-ui-settings-options.png[]
 * Visualization: Select options for viewing data collections
 * Hidden: View or edit a list of records that have been hidden from search results.
 
+==== ${catalog-ui} User Profile
+
+When logged in, you can view details about your account using the *User Profile* (image:user-icon.png[user]) menu on the Navigation Bar.
+
+The profile sidebar will display the following information about your user account:
```

Comment:
We are avoiding "you" in the documentation. Maybe "When logged in, a user can view..."